### PR TITLE
[11.0] [MIG] stock_operating_unit

### PR DESCRIPTION
**Pull request by bjeficent**
Superseed #132 
Migration of stock_operating_unit to version 11.0

### CODE REVIEW

File: stock_operating_unit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "name": "Stock with Operating Units",
+    "summary": "An operating unit (OU) is an organizational entity part of a "
+               "company",
+    "version": "11.0.1.0.1",
+    "category": "Generic Modules/Sales & Purchases",
+    "author": "Eficent, "
+              "Serpent Consulting Services Pvt. Ltd., "
+              "Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "website": "http://www.eficent.com",
```
Comment (Nikul-Chaudhary) [inline stock_operating_unit/__manifest__.py]: @bjeficent URL should be https://github.com/OCA/operating-unit

File: stock_operating_unit/model/stock_location.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# © 2019 Eficent Business and IT Consulting Services S.L.
+# © 2019 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from openerp import _, api, fields, models
```
Comment (Nikul-Chaudhary) [inline stock_operating_unit/model/stock_location.py]: `odoo` everywhere

**Review by Nikul-Chaudhary — COMMENTED** [review]
Improve code

**Comment by bjeficent** [comment]
Comments attended, thanks for reviewing

**Review by AaronHForgeFlow — COMMENTED** [review]
Dismiss review. It is related to another module. `sale_stock_operating_unit`.

**Review by danypr92 — APPROVED** [review]
:clap: :clap: :clap:

**Review by enricostano — APPROVED** [review]

**Pull request merged**